ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.15-nightly-x86.yaml

```diff
@@ -81,6 +87,84 @@ tests:
     - chain: openshift-qe-data-path-tests
     workflow: openshift-qe-installer-azure-ipi-ovn-ipsec
   timeout: 7h0m0s
+- as: control-plane-multi-24nodes
+  cron: 0 4 16 * *
```

Comment:
we have two jobs starting at this same time.

I suggest: '0 10 23 * *'

Reply:
Thanks. Updated.